docs/prover/changelog.md

```diff
@@ -4,35 +4,108 @@ Release Notes
 ```{contents}
 ```
 
+4.12.1 (September 17, 2023)
+---------------------------
+
+### CVL
+- [bugfix] fix to bitwise operations
+- [bugfix] verify range of `nativeBalances[addr]` values
+- [bugfix] no duplication of multidim ghosts with axioms
+- [feat] delete summary qualifiers for faster preprocessing and dealing with analysis-breaking external functions. If a function is never called from spec, it will not be processed. In cases it’s externally called from Solidity, the summary will apply.
+- [feat] greater flexibility of internal summaries - allows accepting as arguments and returning certain reference types: primitive arrays, `bytes`, and structs which may (in nested structs too) contain primitive arrays
+- [feat] support multiple return values from CVL functions
+- [bugfix] Support keywords as struct fields and user defined type names
+- [bugfix] Fix to multi-assert mode in the case multiple CVL asserts in a rule share the same message
+- [ux] Skip rules where all methods are filtered out
+- [bugfix] Do not drop quantifiers when instrumenting vacuity checks
+- [ux] Improved error messages for preserved block errors
+- [bugfix] Support invariant preserved blocks for functions with an argument which is an array of structs
+- [feat] New keyword: executingContract available inside opcode hooks
+- [bugfix] Applying the CALL opcode hook even if the balance transfer fails
+- [bugfix] Support assigning to a wildcard variable
+- [bugfix] Fail if CVL function is non-void and does not end with a return statement
+
+### Modeling (fixes for false-positives)
+- [bugfix] Fix modeling of the length in hash functions
+- [bugfix] Avoid scalarization of directly hashed structs
+- [bugfix] Fix constant propagation due to trusting non-trivial def analysis results to be all dominating
+- [bugfix] Enable heuristical linking in constructors by forcing freemem pointer scalarization
+- [bugfix] Some handling for revert data information passed between calls
```

Comment:
I feel like none of this is useful to our users; I would either focus on the impact to the user or just collapse it into a single bullet point like "fixes for some false positives"

Reply:
that's not very useful either, so I'll just omit the whole section